src/data/feed/quandl.js

```diff
@@ -0,0 +1,146 @@
+        }
+        if (collapse != null) {
+            params.push('collapse=' + collapse);
+        }
+
+        var url = 'https://www.quandl.com/api/v3/datasets';
+        url += ('/' + database + '/' + dataset + '/').replace(/\/\//g, '');
+        url += 'data.json?' + params.join('&');
+
+        d3.json(url, function(error, data) {
+            if (error) {
+                cb(error);
+                return;
+            }
+
+            data = data.dataset_data;
+            var colNames = data.column_names.map(columnNameMap);
+
+            data = data.data.map(function(d) {
+                var output = {};
+
+                output[colNames[0]] = new Date(d[0]);
+                for (var i = 1; i < colNames.length; i++) {
+                    if (colNames[i] != null) {
+                        output[colNames[i]] = d[i];
+                    }
+                }
+
+                return output;
+            });
+
+            cb(error, data);
+        });
+    };
+
+    // Unique Database Code (e.g. WIKI)
+    quandl.database = function(x) {
+        if (!arguments.length) {
+            return database;
+        }
+        database = x;
+        return quandl;
+    };
+    // Unique Dataset Code (e.g. AAPL)
+    quandl.dataset = function(x) {
+        if (!arguments.length) {
+            return dataset;
+        }
+        dataset = x;
+        return quandl;
+    };
+    // Set To Use API Key In Request (needed for premium set or high frequency requests)
+    quandl.apiKey = function(x) {
+        if (!arguments.length) {
+            return apiKey;
+        }
+        apiKey = x;
+        return quandl;
+    };
+    // Start Date of Data Series
+    quandl.start = function(x) {
+        if (!arguments.length) {
+            return start;
+        }
+        start = x;
+        return quandl;
+    };
+    // End Date of Data Series
+    quandl.end = function(x) {
+        if (!arguments.length) {
+            return end;
+        }
+        end = x;
+        return quandl;
+    };
+    // Limit Number of Rows
+    quandl.rows = function(x) {
+        if (!arguments.length) {
+            return rows;
+        }
+        rows = x;
+        return quandl;
+    };
+    // Return Results In Descending Order (true) or Ascending (false)
+    quandl.descending = function(x) {
+        if (!arguments.length) {
+            return descending;
+        }
+        descending = x;
+        return quandl;
+    };
+    // Periodicity of Data (daily | weekly | monthly | quarterly | annual)
+    quandl.collapse = function(x) {
+        if (!arguments.length) {
+            return collapse;
+        }
+        collapse = x;
+        return quandl;
+    };
+    // Function Used to Normalise the Quandl Column Name To Field Name, Return Null To Skip Field
+    quandl.columnNameMap = function(x) {
+        if (!arguments.length) {
+            return columnNameMap;
+        }
+        columnNameMap = x || _columnNameMap;
```

Comment:
I'm not keen on the following behaviour:

```
foo.setBar('baz') !== foo.getBar();
```

I think it is confusing for the user. Can you fall-back to your default implementation in the component function?